index.d.ts

```diff
@@ -89,7 +89,8 @@ declare namespace parser {
 declare const parser: {
     defaultOptions: parser.DefaultOptions;
     defaultDirectives: parser.Directive[];
-
+    
+    (options?: parser.Options): (content: string) => parser.Tree;
```

Comment:
https://github.com/posthtml/posthtml-parser/blob/9327c7e347727cd5d1a5a95cfdbca0c01932bfd7/index.js#L136-L143

According to the code, you must always pass options: 

```ts
options? → options
```

Is this correct behavior or a bug? Why do we always need to pass an object?

//cc @voischev @Scrum

Reply:
Must transmit because be function scope https://github.com/posthtml/posthtml-parser/blob/9327c7e347727cd5d1a5a95cfdbca0c01932bfd7/index.js#L129-L143